Source/SwiftLintFramework/Rules/RuleConfigurations/ExplicitTypeInterfaceConfiguration.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  ExplicitTypeInterfaceConfiguration.swift
+//  SwiftLint
+//
+//  Created by Rounak Jain on 2/18/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+private extension SwiftDeclarationKind {
+    init?(excludedVar: String) {
+        switch excludedVar {
+        case "instance":
+            self = .varInstance
+        case "local":
+            self = .varLocal
+        case "static":
+            self = .varStatic
+        case "class":
+            self = .varClass
+        default:
+            return nil
+        }
+    }
+}
+
+public struct ExplicitTypeInterfaceConfiguration: RuleConfiguration, Equatable {
+    public var severityConfiguration = SeverityConfiguration(.warning)
+
+    var allowedKinds: Set<SwiftDeclarationKind> = [.varInstance,
+                                                   .varLocal,
+                                                   .varStatic,
+                                                   .varClass]
+
+    public var consoleDescription: String {
+        return severityConfiguration.consoleDescription + ", allowed kinds: \(allowedKinds)"
```

Comment:
can we print the allowedKinds in the same short style that is expect to be in the configuration file?